stores/GlobalPath/GlobalPathTypes.ts

```diff
@@ -0,0 +1,13 @@
+export type WayPoint = {
+    latitude: Number,
+    longitude: Number
+}
+
+export type GlobalPath = {
+    GlobalPath: WayPoint[];
```

Comment:
The field name should still be `waypoints` here, just wanted to change the type name